src/app/routes/topic/index.js

```diff
@@ -1,11 +1,10 @@
 import { TopicPage } from '#pages';
 import { topicPath } from '#app/routes/utils/regex';
 import { TOPIC_PAGE } from '#app/routes/utils/pageTypes';
-import isLive from '#app/lib/utilities/isLive';
 import getInitialData from './getInitialData';
 
 export default {
-  path: [topicPath, !isLive() && '/persian/afghanistan'],
+  path: [topicPath && '/persian/afghanistan'],
```

Comment:
Ah... this won't work 😿 

The path needs to be an array of values, containing the regex for all topics (`topicPath`) as well as the "regex" (but really it's just hardcoded) for the /persian/afghanistan page)
 
```suggestion
  path: [topicPath, '/persian/afghanistan'],
```